docs/excel/excel-add-ins-shapes.md

```diff
@@ -1,14 +1,17 @@
 ---
 title: Work with shapes using the Excel JavaScript API
 description: ''
-ms.date: 09/03/2019
+ms.date: 01/07/2020
 localization_priority: Normal
 ---
 
 # Work with shapes using the Excel JavaScript API
 
 Excel defines shapes as any object that sits on the drawing layer of Excel. That means anything outside of a cell is a shape. This article describes how to use geometric shapes, lines, and images in conjunction with the [Shape](/javascript/api/excel/excel.shape) and [ShapeCollection](/javascript/api/excel/excel.shapecollection) APIs. [Charts](/javascript/api/excel/excel.chart) are covered in their own article, [Work with charts using the Excel JavaScript API](excel-add-ins-charts.md).
 
+The following image shows shapes which form a thermometer.
+![Image of a thermometer made as an Excel shape](../images/tempimg.png)
```

Comment:
I think you need a 1px gray border around this image.

Reply:
Nope, not according to newest rules from MSFT image folks - I was surprised by this too

Reply:
FYI This change is due to accessibility requirements. We need images to work on both light and dark background as well as high-contrast. So the gray border is unnecessary and in some cases, might actually throw an error in FastPass (aka Accessibility Insights).